google_metadata_script_runner/main_test.go

```diff
@@ -178,7 +178,7 @@ func TestGetMetadata(t *testing.T) {
 	}
 }
 
-func TestNormalizeTmpFileForWindows(t *testing.T) {
+func TestNormalizeFilePathForWindows(t *testing.T) {
 	tmpFilePath := "C:/Temp/file"
 
 	testCases := []struct {
```

Comment:
go/go-style/best-practices#t-field-labels

Can you please add the field labels to make the test more readable?